packages/recomponents/src/components/r-button/r-button.story.js

```diff
@@ -31,7 +31,7 @@ storiesOf('Components/Button', module)
                 default: boolean('Disabled', false),
             },
             loading: {
-                default: boolean('Disabled', false),
+                default: boolean('Loading', false),
```

Comment:
Loading is fixed by me, see PR: https://github.com/Rebilly/rebilly-recomponents/pull/31

Reply:
Ok, going to cancel this one